### PR TITLE
Update sidebar title[CN]

### PR DESCRIPTION
**Pull request by isdanni**
### Changes

1. Add `yarn` installation for Linux and Windows users in README.md.
2. Update titles from EN to CN in `./website/zh/docs/develop/..` so they appear to be Chinese in the sidebar.
```bash
├── ack
│   └── acknowledgments.md
├── advanced
│   ├── differentiable_programming.md
│   ├── layout.md
│   ├── meta.md
│   ├── odop.md
│   ├── offset.md
│   ├── performance.md
│   ├── sparse.md
│   └── syntax_sugars.md
├── api
│   ├── arithmetics.md
│   ├── atomic.md
│   ├── field.md
│   ├── matrix.md
│   ├── scalar_field.md
│   ├── snode.md
│   ├── ti.md
│   └── vector.md
├── basic
│   ├── external.md
│   ├── field_matrix.md
│   ├── syntax.md
│   └── type.md
├── legacy
│   └── legacy_installation.md
├── misc
│   ├── cli_utilities.md
│   ├── debugging.md
│   ├── export_kernels.md
│   ├── export_results.md
│   ├── extension_libraries.md
│   ├── global_settings.md
│   └── gui.md
└── overview
    ├── hello.md
    ├── install.md
    └── overview.md
```

### CODE REVIEW

**Review by rexwangcc — COMMENTED** [review]
Thank you so much for updating them! I left a few comments. This is a great start towards to migration, I think the next step for us is to look into how to programmatically use the PO files, as it's discussed in #17

File: website/zh/docs/develop/documentation/basic/syntax.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Kernels and functions
+# 内核和方程
```
[the remark this answers is not in the document]
Comment (isdanni) [reply website/zh/docs/develop/documentation/basic/syntax.md:1]: My bad ;) actually hesitated a bit when writing this as I remember vaguely there's a discussion about this. Thanks for pointing out!

**Comment by isdanni** [comment]
> Thank you so much for updating them! I left a few comments. This is a great start towards to migration, I think the next step for us is to look into how to programmatically use the PO files, as it's discussed in #17

Yes exactly what I was thinking! I was going to update one documentation page but figured it might be too much repetitive work if we have to simply copy and paste. Will look into this.

**Review by rexwangcc — APPROVED** [review]
LGTM!

**Pull request merged**